cmd/k8s-operator/deploy/chart/values.yaml

```diff
@@ -22,6 +22,16 @@ operatorConfig:
   defaultTags:
     - "tag:k8s-operator"
 
```

Comment:
I wonder whether all of these options should go under `proxyConfig` instead. If we ever split the operator and API proxy into two separate workloads, it would be the API proxy that would want to keep these options. However, I'd also be a bit worried about possibly breaking changes if people are depending on that in unexpected ways.

Reply:
> I wonder whether all of these options should go under proxyConfig instead

I don't think it should go under `proxyConfig`  that would imply that it gets applied to all other proxies too, which we don't want to do at this point.

I think we don't know what the HA for API server proxy would look like at this point, it could be either:
1) we add a second non-default option where it's split out and that option supports HA
2) we split the API server proxy out as default and that's breaking for everyone (hostname change)
3) we implement HA without splitting API server proxy out (leader election for operator)

It feels like right now we cannot make a decision about where to put the port to optimise for future changes, so under `operatorConfig` is at least consistent with all the other cluster config opts that also apply to the API server proxy (node affinity, labels etc)?

Reply:
Oops I didn't pay enough attention and was thinking `proxyConfig` == `apiProxyConfig`, yeah 100% do not want it in `proxyConfig` :)

I agree the future path for HA'ing the API server proxy is unclear. I think I agree that `operatorConfig` is best for now. It makes sense today and we can plan whatever deprecations/migrations we need in future 👍